indico/modules/events/abstracts/templates/forms/abstract.html

```diff
@@ -0,0 +1,10 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_fieldset %}
```

Comment:
Are we sure we don't want fieldsets though? The abstract creation form is growing pretty long: http://imgur.com/11y9qPW
Maybe a fieldset for the attachments and/or the contribution fields?